tests/ui/components/navigation/navigationcomponent.js

```diff
@@ -109,8 +109,15 @@ describe('navigation component configuration', () => {
     return {
       globalStorage: {
         getState: (storageKey) => {
-          expect(storageKey).toEqual(StorageKeys.VERTICAL_PAGES_CONFIG);
-          return verticalsConfig;
+          switch (storageKey) {
+            case StorageKeys.REFERRER_PAGE_URL:
```

Comment:
could you make the default return null and only add a case for StorageKeys.VERTICAL_PAGES_CONFIG?

Reply:
Definitely could, but then there is no point for the expect line right?
```
expect(storageKey).toEqual(StorageKeys.VERTICAL_PAGES_CONFIG);
```

What do you think?

Reply:
yeah definitely don't need it